crates/kornia-pnp/src/epnp.rs

```diff
@@ -20,9 +24,10 @@ impl PnPSolver for EPnP {
         points_world: &[[f32; 3]],
         points_image: &[[f32; 2]],
         k: &[[f32; 3]; 3],
+        distortion: Option<&kornia_imgproc::calibration::distortion::PolynomialDistortion>,
```

Comment:
does it have to be a ref ? i guess this is a cheap struct to move

Reply:
Done